bookshelf_manager.py

```diff
@@ -0,0 +1,29 @@
+from bookshelf import Book
+from bookshelf import BookShop
+
+
+def main():
+    song_of_ice_and_fire = Book("Song of Ice and Fire", "J. Martin", 400, 200, 1000, 250)
+    harry_potter = Book("Harry Potter", "Joanne Rowling", 400, 100, 8000, 4000)
+    jojo_bizarre_adventures = Book("JoJo Bizarre Adventures", "H. Araki", 900, 300, 10000, 8000)
+
+    bookshop = BookShop()
+    bookshop.add_book(song_of_ice_and_fire)
+    bookshop.add_book(harry_potter)
+    bookshop.add_book(jojo_bizarre_adventures)
+
+    get_top_price_books = bookshop.get_top_books_by_price(2)
+    get_top_sales_books = bookshop.get_top_books_by_sales(2)
+
+    print("\nTop books by price:")
+    for book in top_price_books:
```

Comment:
Undefined variable 'top_price_books' (maybe you mean "get_top_price_books")
if you change the name of the variable above, code will be work